src/main/java/rob/Ui.java

```diff
@@ -0,0 +1,135 @@
+package rob;
+
+/**
+ * Manages the user interface of the chatbot, including the text printed.
+ */
+public class Ui {
+    private static final String GREET = "Hello! I'm Rob! What can I do for you?\n"
+            + "List of commands\n"
+            + "  - todo (desc)\n"
+            + "  - deadline (desc) /by (deadline)\n"
+            + "  - event (desc) /from (start) /to (end)\n"
+            + "  - list\n"
+            + "  - delete (task number)\n"
+            + "  - mark (task number)\n"
+            + "  - unmark (task number)\n"
+            + "  - exit\n";
+    private static final String EXIT = "Bye. Hope to see you again soon!";
+
+    /**
+     * Prints the error message if no file is loaded.
+     *
+     */
+    public void showLoadingError() {
+        System.out.println("No file loaded.");
+    }
+
+    /**
+     * Returns the greeting message shown at the launch of chatbot
+     *
+     *   @return The greeting message string.
+     */
+    public String getGreet() {
+        return GREET;
+    }
+
+    /**
+     * Returns the exit message whenexit command is called
+     *
+     * @return The exit message string.
+     */
+    public String getExit() {
+        return EXIT;
+    }
+
+    /**
+     * Returns the tasks in the provided task list as a string.
+     *
+     * @param taskList The task list containing tasks to be displayed.
+     * @return The list of tasks as a string.
+     */
+    public String showList(TaskList taskList) {
+        StringBuilder list = new StringBuilder();
+        for (int i = 0; i < taskList.len(); i++) {
+            list.append(i + 1).append(". ").append(taskList.getTask(i)).append("\n");
+        }
+        return "____________________________________\n"
+                + list
+                + "____________________________________\n";
+    }
+
+    /**
+     * Returns the delete message.
+     *
+     * @param taskList The task list containing tasks to be deleted.
+     * @param taskNum The task number of the task to be deleted.
+     * @return The deleted task message.
+     */
+    public String delete(TaskList taskList, int taskNum) {
+        // echo
+        String echo = "____________________________________\n"
+                + "deleted: " + taskList.getTask(taskNum - 1) + "\n"
+                + "____________________________________\n";
```

Comment:
I like that you break before the operator! It seems like many people have missed that as part of the coding standard. Nice! 👍